src/test/java/com/g4s8/ghman/user/GhThreadTest.java

```diff
@@ -32,11 +34,48 @@
     @Test
     void parseId() {
         final String tid = "4";
-        MatcherAssert.assertThat(
+        new Assertion<>(
+            "Failed to get thread id from json",
```

Comment:
@olenagerasimova the preferred wording for assertions is the positive one, so here it should be "must get thread id from json".Could you update this one and the other tests here?